contrib/babelfishpg_tsql/runtime/functions.c

```diff
@@ -1844,6 +1845,20 @@ language(PG_FUNCTION_ARGS)
 	PG_RETURN_VARCHAR_P(get_language());
 }
 
+Datum
+identity_into(PG_FUNCTION_ARGS)
+{
+	int64 result;
+	if(tsql_select_into_seq_oid != InvalidOid){
+		result = nextval_internal(tsql_select_into_seq_oid, false);
+	}
+	else{
+		result = 0;
```

Comment:
We should throw an runtime error here or add a Assert(false) here, since I think execution would never be here.